lesson_05/kaliuzhna_05.py

```diff
@@ -0,0 +1,80 @@
+# 1. Inform user that such currencies pair is not supported yet.
+# 2. Be sure that `currency_from_val` positive integer or float
+#    2.1 Inform user that such value is incorrect
+# 3. New function which adds new pairs with ratio to `currencies_map`:
+#    function name: update_currencies_map
+#    with params: currency_from, currency_to, currency_from_val
+#    Inform user about results
+
+
+
+currencies_map = {
+    "USD_UAH": 40.7,
+    "UAH_USD": 0.027,
+}
+
+
+def currency_converter(currency_from, currency_to, currency_from_val):
+    counter = 0
+    if type(currency_from) == str:
+        if currency_from.isalpha():
+            if len(currency_from) == 3:
+                if currency_from.upper() == currency_from:
```

Comment:
probably we should allow to pass `currency_from` in lower case ) just transform it to upper to work with dictionary